packages/onchainkit/src/identity/utils/getAvatar.ts

```diff
@@ -1,71 +1,36 @@
-import { getChainPublicClient } from '@/core/network/getChainPublicClient';
-import { isBase } from '@/core/utils/isBase';
-import { isEthereum } from '@/core/utils/isEthereum';
-import type {
-  Basename,
-  GetAvatarParams,
-  GetAvatarReturnType,
-} from '@/identity/types';
+import type { GetAvatarParams, GetAvatarReturnType } from '@/identity/types';
```

Comment:
At some point, minikit name and avatar will be fetched for all users via an API. To facilitate future additions, I added atomic functions for avatar and name (base and ENS). These functions are consumed by the final `getAvatar` and `getName`